### ISSUE TITLE
Logo Accessibility: wrong usage of title attribute in link tag and missing alt attribute on image tag

### ISSUE THREAD

**Issue by audrasjb** [WordPress/twentytwenty#21]
In function `twentytwenty_get_custom_logo()` located in `functions.php`, there is two accessibility issues:
- `title` attribute should be removed from the link
- an `alt` attribute should be added in the image tag

I'd propose to use the blog title as an `alt` attribute since this is the logo of the website.
